data/src/main/java/org/oppia/data/backends/gae/OppiaGaeClient.kt

```diff
@@ -0,0 +1,26 @@
+package org.oppia.data.backends.gae
+
+import okhttp3.OkHttpClient
+
+import retrofit2.Retrofit
+import retrofit2.converter.moshi.MoshiConverterFactory
+
+object OppiaGaeClient {
+
+  private var retrofit: Retrofit? = null
+
+  val retrofitInstance: Retrofit?
+    get() {
+      if (retrofit == null) {
```

Comment:
Rather than having a var store retrofit, could we use lazy initialization and keep the field as a value?

See: https://www.baeldung.com/kotlin-lazy-initialization.

Reply:
Solved. Made use of kotlin-lazy as per Ben's suggestion.